packages/cdktf-cli/bin/cmds/output.ts

```diff
@@ -6,13 +6,13 @@ import { Errors } from "../../lib/errors";
 const config = cfg.readConfigSync();
 
 class Command implements yargs.CommandModule {
-  public readonly command = "output [stack] [OPTIONS]";
-  public readonly describe = "Prints the output of a stack";
+  public readonly command = "output [OPTIONS] <stacks..>";
+  public readonly describe = "Prints the output of stacks";
   public readonly aliases = ["outputs"];
 
   public readonly builder = (args: yargs.Argv) =>
     args
-      .positional("stack", {
+      .positional("stacks", {
         desc: "Get outputs of stack which matches the given id only. Required when more than one stack is present in the app",
```

Comment:
This can be updated now, as it is not required anymore for multiple stacks, right?